src/client/components/ui/Form/Combobox/Combobox.test.tsx

```diff
@@ -0,0 +1,1154 @@
+			expect(input).toHaveAttribute('aria-activedescendant', option.id);
+			expect(option).toHaveAttribute('aria-selected', 'true');
+		});
+		it('ignore la casse quand les options sont filtrées', async () => {
+			const user = userEvent.setup();
+			render(
+				<Combobox aria-label='Test'>
+					<Combobox.Option>ABC</Combobox.Option>
+					<Combobox.Option>abc</Combobox.Option>
+					<Combobox.Option>Abc</Combobox.Option>
+				</Combobox>,
+			);
+
+			const input = screen.getByRole('combobox');
+			await user.type(input, 'a');
+
+			const options = screen.getAllByRole('option');
+			expect(options).toHaveLength(3);
+		});
+	});
+
+	it('change la valeur de l’input quand on clique sur une option', async () => {
+		const user = userEvent.setup();
+		render(
+			<Combobox aria-label='Test'>
+				<Combobox.Option>Option 1</Combobox.Option>
+				<Combobox.Option>Option 2</Combobox.Option>
+				<Combobox.Option>Option 3</Combobox.Option>
+			</Combobox>,
+		);
+
+		const input = screen.getByRole('combobox');
+		await user.type(input, 'Option');
+		const option = screen.getByRole('option', { name: /Option 2/i });
+		await user.click(option);
+
+		expect(input).toHaveValue('Option 2');
+	});
+	it('ferme la liste quand on clique sur une option', async () => {
+		const user = userEvent.setup();
+		render(
+			<Combobox aria-label='Test'>
+				<Combobox.Option>Option 1</Combobox.Option>
+				<Combobox.Option>Option 2</Combobox.Option>
+				<Combobox.Option>Option 3</Combobox.Option>
+			</Combobox>,
+		);
+
+		const input = screen.getByRole('combobox');
+		await user.type(input, 'Option');
+		const option = screen.getByRole('option', { name: /Option 2/i });
+		await user.click(option);
+
+		const suggestions = screen.getByRole('listbox', { hidden : true });
+		expect(suggestions).not.toBeVisible();
+	});
+	it('affiche un bouton qui déplie le menu', async () => {
+		const user = userEvent.setup();
+		render(
+			<Combobox aria-label='Test'>
+				<Combobox.Option>Option 1</Combobox.Option>
+				<Combobox.Option>Option 2</Combobox.Option>
+				<Combobox.Option>Option 3</Combobox.Option>
+			</Combobox>,
+		);
+
+		const button = screen.getByRole('button');
+		await user.click(button);
+
+		const suggestions = screen.getByRole('listbox');
+		expect(suggestions).toBeVisible();
+	});
+	it('affiche un bouton qui replie le menu quand le menu est déplié', async () => {
+		const user = userEvent.setup();
+		render(
+			<Combobox aria-label='Test'>
+				<Combobox.Option>Option 1</Combobox.Option>
+				<Combobox.Option>Option 2</Combobox.Option>
+				<Combobox.Option>Option 3</Combobox.Option>
+			</Combobox>,
+		);
+
+		const button = screen.getByRole('button');
+		await user.click(button);
+		await user.click(button);
+
+		const suggestions = screen.getByRole('listbox', { hidden: true });
+		expect(suggestions).not.toBeVisible();
+	});
+	it('enlève le bouton qui déplie la liste du tab order', async () => {
```

Comment:
J'ai du lire le test pour comprendre le it :/

suggestion: 'Ne focus pas le bouton qui déplie la liste lorsque je click sur l'input et que je fais tab'

Reply:
'ne focus pas le bouton avec tab' ?

Reply:
oui parfait